packages/core/src/provider.ts

```diff
@@ -11,8 +9,8 @@ export interface ReleaseDeposit {
   amount: number;
 }
 
-export interface ProviderAPI {
-  addDeposit: (account?: string, amount?: BigNumber) => Promise<void>;
+export interface ProviderAPI<T> {
+  addDeposit: (amount?: T, account?: string) => Promise<void>;
```

Comment:
swapping these out so that you can leave account off (and use default of the one that is signed in), but still customize the amount